src/orch/controller.c

```diff
@@ -0,0 +1,83 @@
+#include "controller.h"
+
+#include <errno.h>
+#include <netinet/in.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/socket.h>
+
+int create_master_socket(int port) {
```

Comment:
The port number isn't changed by this function, so this variable should be made const:
`int create_master_socket(const int port)`

Reply:
Changed.